GitHubReleases.ps1

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Get-GitHubRelease
+{
+<#
+    .SYNOPSIS
+        Retrieves information about a release or list of releases on GitHub.
+
+    .DESCRIPTION
+        Retrieves information about a release or list of releases on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER ReleaseId
+        Specific releaseId of a release.
+        This is an option parameter which can limit the results to a single release.
+
+    .PARAMETER Latest
+        Retrieve only the latest release.
+        This is an optional parameter which can limit the results to a single release.
+
+    .PARAMETER Tag
+        Retrieves a list of releases with the associated tag.
+        This is an optional parameter which can filter the list of releases.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Get-GitHubRelease -OwnerName
+
+        Gets all releases for the current owner/repository.
+
+    .EXAMPLE
+        Get-GitHubRelease -ReleaseId 12345
+
+        Get
+
+    .EXAMPLE
+        Get-GitHubRelease -OctoCat OctoCat
+
+    .EXAMPLE
+        Get-GitHubRelease -Uri https://github.com/PowerShell/PowerShellForGitHub
+
+    .EXAMPLE
+        Get-GitHubRelease -OrganizationName PowerShell
+
+#>
```

Comment:
Please add a `.NOTE` section with this statement:

`Information about published releases are available to everyone. Only users with push access will receive listings for draft releases.`